addons/kestrel4500/functions/fnc_updateMemory.sqf

```diff
@@ -0,0 +1,20 @@
+/*
+ * Author: Ruthberg
+ * Update Memory of Kestrel
+ *
+ * Arguments:
+ * 0: Slot <NUMBER>
+ * 1: Value <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [1, "test"] call ace_kestrel4500_fnc_updateMemory
+ *
+ * Public: No
+ */
+params ["_slot", "_value"];
```

Comment:
* Missing include
* Probably enter some empty line between params and the header.
* I don't see any reason for this code to be a global function, it could very well stay within the file where it came from, as it's not being used anywhere else apart from in `fnc_collectData.sqf`